src/main/java/com/example/onlinebookstore/repository/BookRepository.java

```diff
@@ -7,4 +7,6 @@ public interface BookRepository {
     Book save(Book book);
 
     List<Book> findAll();
+
+    Book getById(Long id);
```

Comment:
I think we need to use Optional return type to be sure we will not get NPE if the necessary book is not found.